tests/test_microtiles.py

```
@@ -0,0 +1,462 @@
+        Perturbation size for finite difference evaluation. Defined in conftest.py
+    n_test_points: int
+        Number of testing points in the parametric domain. Defined in conftest.py
+    fd_derivative_stepsizes_and_weights: dict<int:int>
+        Stepsizes and weights for finite difference scheme. Defined in conftest.py
+    """
+    tile_creator = tile_class()
+    # Skip test if tile class has no implemented sensitivities
+    if not tile_creator._sensitivities_implemented:
+        skip(f"Tile {tile_class.__name__} has no sensitivities implemented")
+
+    def generate_random_parameters(tile_creator, np_rng):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `generate_random_parameters`

(ANN202)